x-pack/test/detection_engine_api_integration/security_and_spaces/tests/generating_signals.ts

```diff
@@ -0,0 +1,508 @@
+/*
```

Comment:
Please don't forget to expand this if you don't see github doing for this PR. This is where newly created signal end to end tests I added are.